benchmark/bandwagon_settings.py

```diff
@@ -0,0 +1,26 @@
+"""
+Settings for bandwagon alpha.
+
+Idea:
+    We believe the 'wind' on the market.
+    The wind blow if some instruments rise/fall in continuous x day,
```

Comment:
The wind blow`s` when the instrument rises/falls for consecutive x days, and accumulated change rate is more than y.